8Knot/assets/color.css

```diff
@@ -1,21 +1,45 @@
-h2,
-h4,
-h5,
-h6 {
-  color: #b5b683 !important;
+/* ==== CSS VARIABLES FOR CONSISTENT THEMING ==== */
+:root {
+  --color-primary: #6b8976;
+  --color-secondary: #b5b683;
+  --color-accent: #8d9d92;
+  --color-muted: #aaa;
+  --color-white: #ffffff;
+  --color-black: #000000;
```

Comment:
Love the refactoring of colors into one place up top for easy reuse/theming changes

Reply:
+1